packages/web/src/pages/something-wrong/SomethingWrong.tsx

```diff
@@ -21,14 +23,22 @@ const messages = {
 
 const INVALID_BACK_PAGES = new Set([ERROR_PAGE, SIGN_IN_PAGE, SIGN_UP_PAGE])
 
+const emojiMap: Record<UiErrorCode, React.ReactNode> = {
+  [UiErrorCode.UNKNOWN]: <i className='emoji xl heavy-black-heart' />,
+  [UiErrorCode.RELAY_BLOCKED]: <i className='emoji xl confused-face' />
+}
+
 export const SomethingWrong = () => {
   const lastLocation = useLastLocation()
   const isMobile = useIsMobile()
   const dispatch = useDispatch()
+  // Select only once per mount
+  const uiErrorCode = useSelector(getUiErrorCode, () => true)
```

Comment:
is the equality fn needed?

Reply:
without it, further errors could change the UI error code and make the emoji flicker/change. This equality makes it so it only uses the emoji error code on first render.